cfgov/jinja2/v1/_includes/organisms/header.html

```diff
@@ -1,5 +1,41 @@
 <header class="o-header"
         role="banner">
+
+    {% if flag_enabled('BETA_NOTICE') %}
+    <div class="m-global-banner">
```

Comment:
Because the `m-global-banner` is no longer on the same div as the expandables, some of the LESS needs to change. Remove the `&` on L15 of what used to be beta.less.